components/aws/sagemaker/train/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
BTW, do we want to create separate containers for different components? That would be efforts to maintain build process

Reply:
@mbaijal ?

Reply:
if same containers works then why do we have to keep a separate containers?
I believe this is just a launchpad so does it really matter?

Reply:
Will use a single container for now